hotkeys.py

```diff
@@ -97,17 +88,23 @@ def get_context(app):
             'cwd': cwd
             }
 
+def main_loop():
+    print(f'{HIDE_CURSOR}')
+    # TODO better handle context change
+    last_active_id = None
+    last_cwd = ''
+    while True:
+        try:
+            active_app = NSWorkspace.sharedWorkspace().activeApplication()
+            context = get_context(active_app)
+            if context['id'] != 'hotkeys' and (context['id'] != last_active_id or context['cwd'] != last_cwd):
+                last_active_id = context['id']
+                last_cwd = context['cwd']
+                on_context_change(context)
+            sleep(1)
+        except KeyboardInterrupt:
+            print(f'{SHOW_CURSOR}')
+            sys.exit(0)
```

Comment:
`ctrl+c` is the normal case.

Is there a good way to detect 'on exit script'?

I was initially using signal handling to catch `SIGINT` but I would like to be able to handle `SIGTERM` and crashes as well.

Reply:
Created #3 for this.